### PR TITLE
Organization.qualification extension and ASAM value set(s)

### PR DESCRIPTION
**Pull request by KasiSribal**
ASAM Level Files Uploaded, Please review and approve

### CODE REVIEW

File: input/resources/StructureDefinition-extension-qualification.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "Qualification",
+  "url": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification",
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-extension-qualification.json:4]: change to "url": "http://signalbhn.org/fhir/StructureDefinition/organization-qualification"

File: input/resources/StructureDefinition-extension-qualification.json [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Organization"
+    }
+  ],
+  "type": "Extension",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-extension-qualification.json:27]: change to "baseDefinition": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification"

File: input/resources/StructureDefinition-extension-qualification.json [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+            "code": "Reference"
+          }
+        ]
+      },
+      {
+        "id": "Extension.extension:url",
+        "path": "Extension.extension",
+        "sliceName": "url",
+        "short": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification",
+        "min": 1,
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-extension-qualification.json:201]: Why is `extension:url` a required field?  I am not even sure what this slice is for...  Would you double-check.  Did you get it from here?  
https://build.fhir.org/ig/HL7/fhir-us-ndh/StructureDefinition-base-ext-qualification.html

File: input/vocabulary/valueset-ASAM Difference.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-ASAM Difference.json:1]: What FHIR element will this bind to?

File: input/vocabulary/valueset-ASAM Difference.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "system" : "urn:iso:std:iso:3166",
+        "code" : "US"
+      }]
+    }],
+    "copyright" : "Used by permission of HL7 International, all rights reserved Creative Commons License",
+   "compose" : {
+      "include" : 
+      [
+             {
+          "system": "http://terminology.hl7.org/CodeSystem/",
```
Comment (enjoysparkling) [inline input/vocabulary/valueset-ASAM Difference.json:23]: To use this, we will need to change the system, to create our own.
For now, use:  "http://signalbhn.org/fhir/CodeSystem/asam-difference"

File: input/vocabulary/valueset-ActualASAMLevel.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-ActualASAMLevel.json:1]: What FHIR element will this bind to?

Comment (enjoysparkling) [inline input/vocabulary/valueset-ActualASAMLevel.json:1]: And, I'm sorry I forget.  How is ActualASAM different from PrescribedASAM?  Are they going to be used in the same element?

File: input/vocabulary/valueset-PrescribedASAMLevel.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-PrescribedASAMLevel.json:1]: See notes on ActualASAM

File: input/vocabulary/valueset-ActualASAMLevel.json [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        "system" : "urn:iso:std:iso:3166",
+        "code" : "US"
+      }]
+    }],
+    "copyright" : "Used by permission of HL7 International, all rights reserved Creative Commons License",
+   "compose" : {
+      "include" : 
+      [
+             {
+          "system": "http://terminology.hl7.org/CodeSystem/",
```
Comment (enjoysparkling) [inline input/vocabulary/valueset-ActualASAMLevel.json:23]: for now, change to "system": "http://signalbhn.org/fhir/CodeSystem/asam-actual"
@Gokulsignal - Can we find this code system defined anywhere?  Is there documentation at https://www.asam.org/ that we can use?

File: input/resources/StructureDefinition-extension-qualification.json
Comment (enjoysparkling) [inline input/resources/StructureDefinition-extension-qualification.json:1]: @KasiSribal - add this extension into the [Organization](https://github.com/enjoysparkling/signal-mso-fhir-profiles/blob/main/input/resources/StructureDefinition-profile-organization.json) resource as part of the feature addition.

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
See notes, several changes in these files, modify Organization for the new extension, and add a few research items.

File: input/resources/StructureDefinition-extension-qualification.json (hunk 1 above)
Comment (KasiSribal) [reply input/resources/StructureDefinition-extension-qualification.json]: Changed

File: input/resources/StructureDefinition-extension-qualification.json (hunk 2 above)
Comment (KasiSribal) [reply input/resources/StructureDefinition-extension-qualification.json]: Changed

File: input/resources/StructureDefinition-extension-qualification.json (hunk 3 above)
Comment (KasiSribal) [reply input/resources/StructureDefinition-extension-qualification.json:201]: Yes Matt, This one 
image

**Comment by enjoysparkling** [comment]
I am going to close this PR.  There is still substantial work to do to include this extension (see issue notes).  Once those are complete, open a new PR.

**Pull request closed without merging**